src/DwitTech.NotificationService.WebApi/token_send.json/Google.Apis.Auth.OAuth2.Responses.TokenResponse-user

```diff
@@ -0,0 +1 @@
+{"access_token":"ya29.a0AVvZVspR25JN8tvHn076wqIud2cPekgWYaC4sKrR2fdU5AjewhffoQczCld5poDq6qvao8Uj0Xo3nSLobfDqyetxdzVDyfanEVP45qcHBJFH6bo4RrUHlLELaVQjehdmYEqGXxSrnyYon33HRNoWXftsaD5w7oR6aCgYKAXESAQASFQGbdwaIokmg4nqo9cShrAl0r0Y4qA0167","token_type":"Bearer","expires_in":3599,"refresh_token":"1//03RPtITt9zLsUCgYIARAAGAMSNwF-L9IrRE5r1hAhdJhYcDts3SPGMd4ZMPWcbKpZ-7wz-xnYm7gwUzUPXi3VhKpFoISwuuveSgk","scope":"https://www.googleapis.com/auth/gmail.send","Issued":"2023-02-18T08:23:13.136+05:30","IssuedUtc":"2023-02-18T02:53:13.136Z"}
```

Comment:
Secrets in your code., Please remove urgently

Reply:
Okay boss.